internal/printer/print-to-tsx.go

```diff
@@ -137,21 +137,57 @@ var htmlEvents = map[string]bool{
 	"onwheel":                   true,
 }
 
-func getScriptTypeForNode(n Node) string {
-	if n.Attr == nil || len(n.Attr) == 0 {
+func getStyleLangFromAttrs(attrs []astro.Attribute) string {
+	if len(attrs) == 0 {
+		return "css"
+	}
+
+	for _, attr := range attrs {
+		if attr.Key == "lang" {
+			if attr.Type == astro.QuotedAttribute {
+				return strings.TrimSpace(strings.ToLower(attr.Val))
+			} else {
+				// If the lang attribute exists, but is not quoted, we can't tell what's inside of it
+				// So we'll just return "unknown" and let the downstream client decide what to do with it
+				return "unknown"
```

Comment:
Does this mean if the user writes `<style lang=scss>`, we return `"unknown"`? Shouldn't it be possible to parse the value still per HTML behaviour?

Reply:
Ah yeah, you're right. I always forget about unquoted values, will fix!

Reply:
It turns out that for our compiler, unquoted values are included in the `QuotedAttribute` type, so no code change needed here!